Autopilot/Path Manager/PathManager.c

```diff
@@ -667,12 +667,37 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+// check if power on reset, brown out reset, watchdog time out or software reset flags are set
+
+ char checkForReset(){
+     if (RCONbits.POR || RCONbits.BOR || RCONbits.WDTO || RCONbits.SWR || RCONbits.EXTR){
+         // clear RCON(Reset control) bits)
+         RCONbits.BOR = 0;
```

Comment:
I see why you're resetting the startup codes. Instead add a flag in the code (a boolean) and in line 654 and 655, the first time you receive one or more satellite fixes, set the flag and run the reset home coordinates function.